runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/BenefitsSection/BenefitsCarusel/BenefitsCarusel.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
```

Comment:
imo we should not import `React` in our components since we have react 18. I remember that there is eslint rule that demands it, is it still present now? If yes then remove it, if no then id don't know what does it to here xd

just import the hook individually